test/NuGet.Core.Tests/NuGet.Packaging.Test/PackageBuilderTest.cs

```diff
@@ -2518,6 +2518,33 @@ var contentTypesReader
             }
         }
 
+        [Fact]
+        public void PackageBuilderWorksWithFilesHavingCurrentDirectoryAsTarget()
```

Comment:
the one added was a valid case to define target as "." but defining "./blah/../foo.txt" or similar stuff should not be advised and we can always ask them to correct these targets. And that's why i didn't add tests around that